From 56fe18f9e0535c56d506b17106dc18be0b64a1a5 Mon Sep 17 00:00:00 2001
From: Tobias Thierer <tobiast@google.com>
Date: Fri, 9 Feb 2018 09:44:50 +0000
Subject: [PATCH 38/47] Revert "Move neon block to arm level for libvpx"

This reverts commit 7744504d293dc784c4de794e4eb9df25b8ff69d8.

Reason for revert: Broke several builds. I'm acting build cop, reverting.

Bug: 70351683

Change-Id: Ia57199d8ad2ad187fe9210d424fa8e566bb4452c
---
 Android.bp    | 12 +++++++++---
 Android.bp.in | 12 +++++++++---
 2 files changed, 18 insertions(+), 6 deletions(-)

diff --git a/Android.bp b/Android.bp
index 950ab3c..7f921c6 100644
--- a/Android.bp
+++ b/Android.bp
@@ -2082,9 +2082,15 @@ cc_library_static {
             // Want arm, not thumb, optimized
             instruction_set: "arm",
 
-            srcs: libvpx_arm_neon_c_srcs,
-            generated_sources: ["libvpx_arm_neon_asm_srcs_converted"],
-            local_include_dirs: ["config/arm-neon"],
+            srcs: libvpx_arm_c_srcs,
+            local_include_dirs: ["config/arm"],
+
+            neon: {
+                exclude_srcs: libvpx_arm_c_srcs,
+                srcs: libvpx_arm_neon_c_srcs,
+                generated_sources: ["libvpx_arm_neon_asm_srcs_converted"],
+                local_include_dirs: ["config/arm-neon"],
+            },
         },
 
         arm64: {
diff --git a/Android.bp.in b/Android.bp.in
index 0ad23b3..ac6a464 100644
--- a/Android.bp.in
+++ b/Android.bp.in
@@ -24,9 +24,15 @@ cc_library_static {
             // Want arm, not thumb, optimized
             instruction_set: "arm",
 
-            srcs: libvpx_arm_neon_c_srcs,
-            generated_sources: ["libvpx_arm_neon_asm_srcs_converted"],
-            local_include_dirs: ["config/arm-neon"],
+            srcs: libvpx_arm_c_srcs,
+            local_include_dirs: ["config/arm"],
+
+            neon: {
+                exclude_srcs: libvpx_arm_c_srcs,
+                srcs: libvpx_arm_neon_c_srcs,
+                generated_sources: ["libvpx_arm_neon_asm_srcs_converted"],
+                local_include_dirs: ["config/arm-neon"],
+            },
         },
 
         arm64: {
-- 
2.17.1

